### ISSUE TITLE
Error in the positioning of distant asteroids

### ISSUE THREAD

**Issue by RobertoW46** [Stellarium/stellarium#3815]
Dear:

Yesterday I did an observation session of classical asteroids and minor planets as for example “1 Ceres”, “68 Leto”, “42 Isis”, “471 Papagena”, “40 Harmonia”, “16 Psyche”, “7 Iris”, “1036 Ganymed”, “20 Massalia”, “39 Laetitia” and ‘11 Parthenope’ among others and I detected that most of them are about 15 minutes of arc above the real position, which makes it difficult to locate them with respect to the nearby asterisms taken as reference.

I take the liberty of commenting on this as a contribution to continue improving this excellent software that is very helpful for those of us who are dedicated to astronomy.

Next, I am sending you the question in the format requested by you:

### Expected Behaviour

That the location of the dwarf panets and classical asteroids found have a real position in the sky.

### Actual Behaviour
Describe or maybe attach a screenshot?

The dwarf panets and classical asteroids found are about 15 arcminutes above the actual position observed in the telescope eyepiece.
I am sending you a screenshot of “1 Ceres” but the other astroids imaged were also not positioned in their correct place:

![1](https://github.com/user-attachments/assets/ed9e5cc1-1889-4ce1-9db8-1092a657b4b5)

(I'm sorry but I can't upload the image because I can't find the way to copy it directly here; if you tell me how to do it, please explain it to me and I will be glad to do it).

### Steps to reproduce

The dwarf panets and classical asteroids found are about 15 arcminutes above the actual position observed in the telescope eyepiece.

Greetings and again a thousand apologies for daring to report the error I saw yesterday, Roberto Wiman.

**Comment by RobertoW46** [comment]
A thousand apologies but by mistake, the comment of the arrow in the screenshot was left in Spanish, so I translate the text: "In the eyepiece of the telescope I saw it about 10 arcminutes below the position the position indicated in Stellarium, which is approximately where
marks the arrow".

**Comment by Atque** [comment]
Are the orbital elements of the asteroids up to date? When selecting an asteroid, does the infostring to the left say anything about outdated orbital element?

**Comment by RobertoW46** [comment]
Thanks for your prompt reply.
Forgive my ignorance but I can't find where I can check if the orbital elements of the asteroids are updated.
The string of information on the left that you refer to is the one that informs about magnitude, distance and diameter among other things?
I'll pass you a screenshot of the location of (68) Leto:

![1](https://github.com/user-attachments/assets/377b5317-46bc-4e72-a7a9-b0e0a76a0a41)

Greetings to all, Roberto.

**Comment by alex-w** [comment]
@RobertoW46 please try to update orbital elements via Solar System Editor plugin and check the positions again

**Comment by RobertoW46** [comment]
¡Gracias por la indicación!
Voy a hacerlo de esa forma y verificar las posiciones para saber si se soluciona el problema cuando se despeje el cielo, ya que estos días están nublados por la zona donde estoy.
Una vez verificado, lo tendré al tanto.

Saludos a todos y nuevamente gracias, Roberto.

**Comment by RobertoW46** [comment]
Thanks for the indication!
I'm going to do it that way and check the positions to see if it fixes the problem when the skies clear, as these days are cloudy around the area where I am.
Once verified, I will keep you posted.

Greetings to all and thanks again, Roberto.

**Comment by gzotti** [comment]
https://github.com/Stellarium/stellarium/wiki/Common-Problems-for-the-current-version#asteroid-positions-inaccurate

**Comment by RobertoW46** [comment]
Hi all:

I did as I was told and updated the orbital elements of the asteroids I chose through the Solar System Editor add-on to verify their positions.
I did it individually for each asteroid because I don't have them all loaded due to it makes it much slower to run the program.
So, even though it was cloudy, I just took out the telescope and found a couple of small windows between the clouds to point to 3 of the asteroids that did not appear well positioned.
The result was optimal because their position was the one indicated by the eyepiece, so I am very grateful to @alex-w and @gzotti.

Regards and I am sorry for having bothered you with such an elementary doubt, Roberto.

Translated with DeepL.com (free version)
